### DISCUSSION TITLE
October 2024 Survey Results - Question 2

### DISCUSSION THREAD

**Discussion by boc-the-git** [mealie-recipes/mealie#4640] [General]
Placeholder discussion for people to share their thoughts and insights about the survey results.

**Comment by ChipSkylark37** [comment]
It was a bit surprising to see the amount of Postgres installations, when the documentation generally recommends SQLLite.

```
Step 1: Deployment Type

SQLite is a popular, open source, self-contained, zero-configuration database that is the ideal choice for Mealie when you have 1-20 Users and your concurrent write operations will be some-what limited.

PostgreSQL might be considered if you need to support many concurrent users. In addition, some features are only enabled on PostgreSQL, such as fuzzy search.

You can find the relevant ready to use docker-compose files for supported installations at the links below.

    [SQLite](https://docs.mealie.io/documentation/getting-started/installation/sqlite/)
    [PostgreSQL](https://docs.mealie.io/documentation/getting-started/installation/postgres/)
```

I am finding that as I access my Mealie instance it takes a few seconds for recipes to load sometimes, or when uploading images I see the black progress bar sitting for a while before it saves. 

There's only 3 users accessing the system, and hardly ever more than 1 person at a time.

I was considering moving to Postgres, if this would help alleviate some of those delays / freezes. 

1. Are there any other features limited by one database vs another?
2. Is Postgres better in terms of "speed" ? 
3. Is there a way to migrate from one to another or would I need to spin up a new instance and do a backup > restore.
